tests/test_utils/amazon_system_helpers.py

```diff
@@ -60,6 +84,25 @@ def execute_with_ctx(cls, cmd: List[str]):
         with provide_aws_context():
             executor.execute_cmd(cmd=cmd)
 
+    @staticmethod
+    def create_connection(aws_conn_id: str,
+                          region: str = "eu-west-1") -> None:
```

Comment:
```suggestion
                          region: str) -> None:
```
Can we let the user decide that? If necessary I would add it as variable (setable via env) to the example dag.

Reply:
I defined default values but not used, user have to define environment variable. You can check PR description and the docstring of ECSSystemTest class. also line 62, 80 and 113 in the airflow/tests/providers/amazon/aws/operators/test_ecs_system.py file.

Reply:
Okay, but then please remove it from here. Or is there a different reason for having this as default value?

Reply:
done :heavy_check_mark:  thanks